### PR TITLE
Added font to mailing attributes

### PR DESCRIPTION
**Pull request by alessio-pani**
Ref: https://dev.azure.com/FarHeapSolutions/Webprints/_backlogs/backlog/Spartans/Backlog%20items/?workitem=188999

### CODE REVIEW

File: src/InkRouter/Models/Attributes/MailingAttributes.php [hunk 1]
```diff
@@ -126,6 +131,24 @@ public function getShipExtra()
...
+    }
+
+    /**
+     * @param string $mailingFont
+     * @return InkRouter_Models_Attributes_MailingAttributes
+     */
+    public function setMailingFont($mailingFont)
+    {
+        $this->mailingFont = $mailingFont;
+        return $this;
```
Comment (fightmaster) [inline src/InkRouter/Models/Attributes/MailingAttributes.php]: add a blank line before return

**Comment by fightmaster** [comment]
👍

**Comment by richardfullmer** [comment]
:+1:

File: src/InkRouter/Models/Attributes/MailingAttributes.php (hunk 1 above)
Comment (alessio-pani) [reply src/InkRouter/Models/Attributes/MailingAttributes.php]: done

**Comment by fightmaster** [comment]
> 👍

I haven't right to merge

**Comment by alessio-pani** [comment]
Can someone else merge it? I don't have rights to merge it either

**Pull request merged**